src/PluginStartupRegistration.cpp

```diff
@@ -0,0 +1,81 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file PluginStartupRegistration.cpp
+
+  @author Vitaly Sverchinsky
+
+**********************************************************************/
+
+#include "PluginStartupRegistration.h"
+
+#include <thread>
+
+#include <wx/log.h>
+#include <wx/app.h>
+
+#include "PluginManager.h"
+#include "PluginDescriptor.h"
+#include "widgets/ProgressDialog.h"
+
+
+PluginStartupRegistration::PluginStartupRegistration(std::vector<std::pair<wxString, wxString>> plugins) : mPlugins(std::move(plugins))
+{
+   mValidator = std::make_unique<AsyncPluginValidator>(*this);
+}
+
+void PluginStartupRegistration::OnInternalError(const wxString& error)
+{
+   wxLogError(error);
```

Comment:
Is this where errors from PluginHost are used at last?  Then fixing the setup of locale in the other process is unimportant after all, and literal English string errors are all right.  But then, where `Translation()` was used, make it `Debug()`.

Reply:
Wasn't aware of that, thanks!